Parser/tokenizer.c

```diff
@@ -1639,6 +1675,7 @@ tok_get(struct tok_state *tok, const char **p_start, const char **p_end)
                     }
                     c = tok_nextc(tok);
                 }
+                char* zeros_end = tok->cur;
```

Comment:
Cat `tok->start` be used instead of a new variable?

Reply:
Not here because we want to highlight only the zeros and `tok->cur` points at the end of the number